README.md

```diff
@@ -12,9 +12,20 @@ For more info on localization, how it works in the Wolvic XR project, and how to
 
 ## Setup instructions
 
-> By default Wolvic will try to download prebuilt GeckoView libraries from [Mozilla's maven repositories](https://maven.mozilla.org/maven2/org/mozilla/geckoview/?prefix=maven2/org/mozilla/geckoview/). Note that after [PR #70](https://github.com/Igalia/wolvic/pull/70) WebXR sessions won't work with those images because that PR introduced a change in the GeckoView protocol that is only available in the `wolvic_release` branch from [this repository](https://github.com/Igalia/gecko-dev/). For additional details on how to use a local GeckoView build check [this section](#dependency-substitutions)
+*GeckoView local substitution.*
 
->  **UPDATE**: use `FIREFOX_103_0_2_RELEASE` instead of `wolvic_release` after [PR #256](https://github.com/Igalia/wolvic/pull/256).
+After [PR #70](https://github.com/Igalia/wolvic/pull/70), WebXR sessions won't work with the prebuilt maven GeckoView libraries because that PR introduced a change in the GeckoView protocol that is only available in the `FIREFOX_103_0_2_RELEASE` branch from [this repository](https://github.com/Igalia/gecko-dev/).
```

Comment:
Pending updating the link to the patches.

Reply:
I've just setup https://github.com/Igalia/wolvic-gecko-patches